Payments/Integration/01_ManageConnection.md

```diff
@@ -0,0 +1,145 @@
+[!!Connections (Settings)](../UserInterface/08a_Connections.md)
+
+# Manage connections to payment service providers (PSP)   
+
+If you want to create a new connection to a payment service provider (PSP), you can buy, depending on the payment service provider, the needed connection in the app store or license it directly at the payment service provider. By default, Actindo supports the following drivers:   
+- Adyen&trade; (available in the app store)
+- Amazon pay&trade; (available in the app store)   
+It works without using the Amazon channel connection, but the Amazon driver must be installed.
+- Stripe&trade; (available in the app store)
+- Paypal&trade;
+- A dummy driver to simulate payments for testing purposes in a sandbox   
+
+The data structure of the connection then provides exactly the data that the PSP needs to exchange data with the Actindo *Payments* module.
+
+In the following, the procedures to create, edit, or disable connections are described. 
+
+
+
+## Create PSP connection
+
+Create a connection to a certain payment provider. As soon you have created a connection, an initial synchronization will be triggered, which will transfer the entire data model of the connected payment service provider to Actindo. 
+
+#### Prerequisites
+
+- The relevant drivers are installed both on the main account and &ndash; if needed &ndash; the sandboxes.
+
+#### Procedure    
+
+*Payments > Settings > Tab CONNECTIONS*   
+
+![Connections](../../Assets/Screenshots/Payments/Settings/Settings.png "[Connections]")
+
+1.  Click the ![Add](../../Assets/Icons/Plus01.png "[Add]") (Add) button in the bottom right corner.   
+    The *Connection* view is displayed.   
```

Comment:
The *Create connection* view is displayed.

Vgl. Omni-Channel: 
https://github.com/Actindo-AG/Documentation/blob/main/Channels/Integration/01_ManageConnections.md

Reply:
Auf dem Screen steht "Connection"

Reply:
Steht zwar nicht auf dem Screen, aber wenn es der Einheitlichkeit dient